dependencies.gradle

```diff
@@ -1,178 +1,238 @@
+        autoValueParcelVersion                : "0.2.5",
+
+        daggerVersion                         : "2.10",
+        retrofitVersion                       : "2.1.0",
+        rxJavaVersion                         : "1.2.4",
+        rxAndroidVersion                      : "1.2.1",
+
+        gsonVersion                           : "2.8",
+        cookiesJarVersion                     : "v1.0.0",
+        networkSamplerVersion                 : "connectionclass:1.0.1",
+
+        okIoVersion                           : "1.11.0",
+        okHttp3Version                        : "3.5.0",
+
+        glideVersion                          : "3.7.0",
+        glideOkHttpVersion                    : "1.4.0",
+
+        crashlyticsVersion                    : '2.6.7',
+
+        branchIOVersion                       : '2.6.0',
+
+        lynxVersion                           : '1.6',
+        stethoVersion                         : '1.5.0',
+        tinyDancerVersion                     : '0.1.0',
+        timberVersion                         : "4.5.1",
+        leakcanaryVersion                     : "1.5",
+
+        zetaUtilsVersion                      : "0.4",
+        zetaEitherVersion                     : "0.3",
+        zetaListItemDividerVersion            : "0.2",
+        zetaHeaderAndFooterAdaptersVersion    : "0.2",
+
+        stickyScrollViewVersion               : "1.0.2",
+
+        metagalacticExpandableImageViewVersion: "0.0.2",
+
+        lottieAnimationVersion                : "1.5.1",
+
+        junitVersion                          : '4.12',
+        robolectricVersion                    : '3.0',
+        assertJVersion                        : '1.7.1',
+        equalsVerifierVersion                 : '1.7.5',
+        mockitoVersion                        : '1.10.19',
+        supportTestRunnerVersion              : '0.4.1',
+        supportTestVersion                    : "0.4.1",
+        espressoVersion                       : '2.2.1',
+
+        aptPluginVersion                      : "1.8",
+        fabricPluginVersion                   : "1.22.1",
+        gradlePluginVersion                   : "3.0.0-alpha4",
+        frodoPluginVersion                    : "0.8.3",
+        gmsServicesPluginVersion              : "3.0.0",
+
+        powerMockitoVersion                   : '1.6.2',
 ]
 
 ext.gradlePlugins = [
+        fabricPlugin      : "io.fabric.tools:gradle:$versions.fabricPluginVersion",
         androidToolsPlugin: "com.android.tools.build:gradle:$versions.gradlePluginVersion",
-        retroLambdaPlugin : "me.tatarka:gradle-retrolambda:$versions.retroLambdaPluginVersion",
         frodoPlugin       : "com.fernandocejas.frodo:frodo-plugin:$versions.frodoPluginVersion",
+        gmsServicesPlugin : "com.google.gms:google-services:$versions.gmsServicesPluginVersion",
+        aptPlugin         : "com.neenbedankt.gradle.plugins:android-apt:$versions.aptPluginVersion",
```

Comment:
Yeah! May be copy paste error again. Anyways now that there is appt2 we should check that out.